### PR TITLE
fix: upgrade lib

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,50 +47,49 @@
   "peerDependencies": {
     "ethers": ">=4.0.45"
```
Comment (Nebulis) [inline package.json:48]: hum why do we have this ? also we moved to v5 :thinking:

File: src/index.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
-export {TitleEscrowFactory} from "./contracts/TitleEscrowFactory";
-export {TradeTrustErc721Factory} from "./contracts/TradeTrustErc721Factory";
-export {TitleEscrowCreatorFactory} from "./contracts/TitleEscrowCreatorFactory";
-export {getTitleEscrowCreatorAddress} from "./config";
+/* eslint-disable camelcase */
+export {
+  TitleEscrow__factory as TitleEscrowFactory,
+  TitleEscrowCreator__factory as TitleEscrowCreatorFactory,
+  TradeTrustERC721__factory as TradeTrustErc721Factory,
```
Comment (Nebulis) [inline src/index.ts:5]: this i sa breakin change, the name changed :)

File: package.json (hunk 1 above)
Comment (HJunyuan) [reply package.json:48]: Ok shall we remove this?

There's a devDependency for v5 [here (line 80)](https://github.com/Open-Attestation/token-registry/pull/27/files/d63ebdc9c99b3c8d1e563266b5c52ffc6686249a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R80).

File: src/index.ts (hunk 2 above)
Comment (Nebulis) [reply src/index.ts:5]: ERC vs Erc

File: package.json (hunk 1 above)
Comment (Nebulis) [reply package.json:48]: The thing is that dev dependencies are not installed, so it won't be available for consumers. I'm not sure of the reason the peer dependency is here though

File: src/index.ts (hunk 2 above)
Comment (HJunyuan) [reply src/index.ts:5]: Yes, the original master branch is exported as `Erc`: https://github.com/Open-Attestation/token-registry/blob/c8a39d2b57b4e6a29bce54a59db436ed424509dc/src/index.ts#L2

Just not too sure why the generated filenames automatically capitalised `ERC`.

Comment (Nebulis) [reply src/index.ts:5]: yes so it's a breking change right ?
- before Erc
- now ERC

I don't mind the change, but if it's a breaking change we must release it that way :)

Comment (rjchow) [reply src/index.ts:5]: wanna smack the typechain people

File: package.json (hunk 1 above)
Comment (HJunyuan) [reply package.json]: As requested, I've bumped the ethers peer dependency from v4.0.45 to v5.0.8

**Review by HJunyuan — APPROVED** [review]

**Pull request merged**